main.tf

```diff
@@ -27,21 +34,25 @@ locals {
 
   user_data = templatefile("${path.module}/templates/cloud-init.cfg", {})
   user_data_script = templatefile("${path.module}/templates/cloud-init.sh", {
-    PROXY_CONFIG   = var.proxy_config
-    DB_USER        = var.kong_database_config.user
-    DB_HOST        = local.db_info.endpoint
-    DB_NAME        = local.db_info.database_name
-    CE_PKG         = var.ce_pkg
-    EE_PKG         = var.ee_pkg
-    PARAMETER_PATH = local.ssm_parameter_path
-    REGION         = var.region
-    VPC_CIDR_BLOCK = var.vpc_cidr_block
-    DECK_VERSION   = var.deck_version
-    MANAGER_HOST   = var.manager_host
-    PORTAL_HOST    = var.portal_host
-    SESSION_SECRET = random_string.session_secret.result
-    KONG_CONFIG    = var.kong_config
+    proxy_config     = var.proxy_config
+    db_user          = var.kong_database_config.user
+    db_host          = local.db_info.endpoint
+    db_name          = local.db_info.database_name
+    ce_pkg           = var.ce_pkg
+    ee_pkg           = var.ee_pkg
+    parameter_path   = local.ssm_parameter_path
+    region           = var.region
+    vpc_cidr_block   = var.vpc_cidr_block
+    deck_version     = var.deck_version
+    manager_host     = var.manager_host
+    portal_host      = var.portal_host
+    session_secret   = random_string.session_secret.result
+    kong_config      = var.kong_config
+    kong_ports       = var.kong_ports
+    kong_ssl_uris    = var.kong_ssl_uris
+    kong_hybrid_conf = var.kong_hybrid_conf
```

Comment:
just curious, why move to lower case?

Reply:
just for aesthetic reasons, I wanted to create a more visible difference in the cloud-init.sh template between terraform variable and shell variables.

Reply:
I found this a bit hard to grok on first glance
```
DB_HOST=${DB_HOST}
DB_NAME=${DB_NAME}
```